File: Texxtoor/Solution/Texxtoor.Portal/Scripts/reader/js/bookmark.js
//-----------------Bookmark
function Bookmark(navigationItem, fragmentCfi, id) {
  this.navigationItem = navigationItem;
  this.fragmentCfi = fragmentCfi;
  this.id = id;
}

Bookmark.prototype = {
  id: null,
  fragmentCfi: null,
  navigationItem: null
};

File: Texxtoor/Solution/Texxtoor.DataLayer/ServiceModel/Bookmark.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace Texxtoor.DataModels.ServiceModel {

  [DataContract]
  public class Bookmark {

    [DataMember]
    public int Id {
      get;
      set;
    }

    [DataMember]
    public int BookId {
      get;
      set;
    }
    
    [DataMember]
    public string FragmentCfi {
      get;
      set;
    }

  }

}
